src/cryptoadvance/specter/templates/includes/file-uploader.html

```diff
@@ -0,0 +1,113 @@
+<!--
+  - Image source and width, as well as button width, height and colour can be set
+
+{# - Usage -
+<file-uploader id="label-uploader" image_src="{{ url_for('static', filename='img/file.svg') }}" image_width="30px" button_width="137px" button_height="50px" button_color="hsl(217, 20%, 35%)">
+	<span slot="description">or upload file</span>
+</file-uploader>
+#}
+
+To get the data from the file upload into your jinja file, subscribe to the `file-upload` event in the js script part:
+
+let textarea = document.getElementById("address_labels_data")
+document.addEventListener('file-upload', e => {
+	let result = e.detail.result;
+	textarea.value = result
+	})
+-->
+
+<template id="file-uploader">
```

Comment:
Would you consider naming it `file-uploader-button`?

Reply:
I'd leave it for now, if it ends up being always a button, then it would be more precise.